react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,40 @@
+import { isArray } from './utils';
+
+export function min(...numbers) {
+  if (numbers.length === 0) {
+    return undefined;
+  }
+
+  if (numbers.length === 1) {
+    if (isArray(numbers[0])) {
+      return Math.min(...numbers[0]);
+    }
+
+    return numbers[0];
+  }
+
+  return Math.min(...numbers);
+}
+
+export function copy(myObject) {
+  if (isArray(myObject)) {
+    return myObject.slice();
+  }
+
+  return { ...myObject };
+}
+
+export function reverseMerge(myArray1, myArray2) {
+  const myReversedArray = copy(myArray2);
+  myReversedArray.push(...myArray1);
+  return myReversedArray;
+}
+
+export function filterAttribs(myObject) {
+  const dontAllowed = ['a', 'b'];
+  return Object.keys(myObject).filter(key => !dontAllowed.includes(key)).reduce((obj, key) => {
+    const myNewObj = obj;
+    myNewObj[key] = myObject[key];
+    return myNewObj;
```

Comment:
even though the logic is right, this is too complex and hard to read. There's an easier way to do this. Hint: user the `spread` operator for objects.